docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+
+Go to the *Management / Security / Users* page:
+
+[role="screenshot"]
+image::security/images/management-builtin-users.jpg["User management screenshot in Kibana"]
+
+In this example, you can see all of the built-in users. 
+
+Click *Create user* to make a new user. For example, create a user for yourself:
+
+[role="screenshot"]
+image::security/images/create-user.jpg["Creating a user in Kibana"]
+
+In <<get-started-elastic-stack>>, you configured {ls} to listen for {metricbeat} 
+input and to send the events to {es}.  You therefore need to create a user 
+that {ls} and {metricbeat} can use to communicate with {es}. Click *Create user* 
+and create a `metricbeat_internal` user, for example:
+
+[role="screenshot"]
+image::security/images/create-metricbeat-user.jpg["Creating a {metricbeat} user in {kib}"]
+
+You'll notice that when you create a user, you can assign it a _role_, such as 
+`kibana_user`. The {stack} uses roles to _authorize_ users. For example, you can 
+authorize a user to perform specific actions (such as read, create, or manage) 
+on specific secured resources (such as indices, aliases, documents, fields, or 
+clusters). 
+
+To help you get up and running, there are built-in roles. Go to the 
+*Management / Security / Roles* page to see them:
+
+[role="screenshot"]
+image::security/images/management-roles.jpg["Role management screenshot in Kibana"]
+
+Select a role to see more information about what privileges it provides. For 
+example, if you select the `kibana_user` role, you will see that it grants 
+`manage`, `read`, `index`, and `delete` privileges on the `.kibana*` indices. 
+
+If you want to learn more about authorization and roles, see <<authorization>>.
+
+[float]
+[[gs-roles]]
+=== Create roles in {kib}
+
+If you completed all of the steps in <<get-started-elastic-stack>>, you should 
+have {metricbeat} data stored in {es}. Let's create two roles that grant 
+different levels of access to that data. 
+
+Go to the *Management / Security / Roles* page and click *Create role*. 
+
+Create a `metricbeat_reader` role that has `read` and `view_index_metadata` 
+privileges on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-reader-role.jpg["Creating a role in Kibana"]
+
+Create a `metricbeat_writer` role that has `manage_index_templates` and `monitor` 
+cluster privileges, as well as `write`, `delete`, and `create_index` privileges
+on the `metricbeat-*` indices:
+
+// TBD: The Beats docs don't mention the need for the delete privilege, whereas 
+// the Logstash docs do. Which should be used in this case?
+
+[role="screenshot"]
+image::security/images/create-writer-role.jpg["Creating another role in Kibana"]
+
+Now go back to the *Management / Security / Users* page and assign these roles 
+to the appropriate users. Assign the `metricbeat_reader` role to your personal 
+user.  Assign the `metricbeat_writer` role to the `metricbeat_internal` user.
```

Comment:
I think it's worth mentioning that the personal user should also have `kibana_user`.

Reply:
You're right, I've revamped this section to clarify that.